documentation/modules/rn-2.5.adoc

```diff
@@ -106,9 +116,9 @@ The error status message for a VM with no operating system on the *Plans* page o
 vSphere only: Migrations from {rhv-short} and OpenStack do not fail, but the encryption key may be missing on the target {ocp} cluster.
 
 
-.Migration fails during precopy/cutover while a snapshot operation is performed on the source VM
+.Migration fails during precopy/cutover while performing a snapshot operation on the source VM
 
-Warm migration from {rhv-short} fails if a snapshot operation is performed on the source VM. If a user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+Warm migration from {rhv-short} fails if the user performs a snapshot operation on the source VM. The migration does not wait for a snapshot operation on the source VM to finish when it is scheduled at the same time as the migration. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
```

Comment:
here changing this to "the user performs" narrows down the cases a bit too much because it might be "the system" that triggers the snapshot (for example, a backup process that runs in the background triggers the snapshot operation). I'd prefer to keep the initiator of the snapshot operation more vague here

something about the second part is not that clear - "it" in "it is scheduled at the same time as the migration" refers to the snapshot operation but the problem doesn't occur when the snapshot operation is scheduled at the same time as the migration but when the migration is scheduled to run while there is an ongoing snapshot operation

Reply:
Changed text